build.gradle

```diff
@@ -82,6 +82,7 @@ subprojects {
     slf4jVersion = '1.7.29'
     commonsLangVersion = '3.12.0'
     logbackVersion = '1.2.5'
+    cyfaceApiVersion = '1.0.0-alpha1' // libs/api
```

Comment:
TO-DO Upgrade to final version after the following PR was merged and version 1.0.0 released:
https://github.com/cyface-de/api/pull/1